testing/trino-benchto-benchmarks/generate_schemas/generate-tpcds.py

```diff
@@ -1,53 +1,63 @@
 #!/usr/bin/env python
```

Comment:
did you use the tool to generate schemas manually as a test?

Reply:
Thanks, running this unveiled a trivial bug.